src/olcs/FeatureConverter.js

```diff
@@ -340,10 +340,14 @@ exports.prototype.olCircleGeometryToCesium = function(layer, feature, olGeometry
  * @param {!Array<Cesium.Cartesian3>} positions The vertices of the line.
  * @return {!Cesium.GroundPrimitive} primitive
  */
-exports.prototype.createStackedGroundCorridors = function(layer, feature, width, color, positions) {
-  let previousDistance = 0;
+exports.prototype.createStackedGroundCorridors = function(width, color, positions) {
+  // Convert positions to an Array if it isn't
+  if (positions[0].constructor.name !== 'Array') {
```

Comment:
This looks inconsistent with the declared type of the `positions` parameter (which is an array of Cartesian3.
I believe you want to change the type to `!Array<Cesium.Cartesian3>|Array<Array<Cesium.Cartesian3>>`

Please also use the standard `Array.isArray()` function.